terraform/vsphere-infra.tf

```diff
@@ -0,0 +1,57 @@
+# Getting vSphere infrastructure
```

Comment:
Maybe nest all these files further under `terraform/vsphere/` to make space for other terraform providers we might want to support in this repo in the future

Reply:
k, def makes sense to me, thanks